tests/integration/integration.go

```diff
@@ -356,7 +356,7 @@ var _ = ginkgo.Describe("[Tx Processing]", ginkgo.Serial, func() {
 			)
 			// Must do manual construction to avoid `tx.Sign` error (would fail with
 			// 0 timestamp)
-			msg, err := tx.Digest()
+			msg, err := tx.UnsignedBytes()
```

Comment:
Can we rename `msg` to `unsignedTxBytes` ?

Reply:
done